ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt

```diff
@@ -142,48 +142,35 @@ hideall "--sync--"
 
 # Phase Three
 488.8 "--sync--" WasDefeated { target: 'Ice Veil' } window 488.8,10
-500.0 "Junction" Ability { id: "9D22", source: "Usurper of Frost" } window 500.0,0
+500.0 "Junction" Ability { id: "9D22", source: "Usurper of Frost" } window 500.0,5
 514.3 "--targetable--"
 518.3 "Hell's Judgment" Ability { id: "9D49", source: "Oracle of Darkness" }
 521.4 "--sync--" Ability { id: "9CB5", source: "Oracle of Darkness" }
 532.4 "Ultimate Relativity" Ability { id: "9D4A", source: "Oracle of Darkness" }
-542.1 "Speed/Quicken/Slow" #Ability { id: "9D65", source: "Oracle of Darkness" }
-544.2 "Dark Fire III" Ability { id: "9D54", source: "Oracle of Darkness" }
-544.2 "Unholy Darkness" #Ability { id: "9D55", source: "Oracle of Darkness" }
-549.3 "Sinbound Meltdown" Ability { id: "9D2B", source: "Delight's Hourglass" }
-551.3 "Sinbound Meltdown (x9)" duration 8.2
-554.0 "Dark Blizzard III" Ability { id: "9D57", source: "Crystal of Darkness" }
-554.0 "Unholy Darkness" Ability { id: "9D55", source: "Crystal of Darkness" }
-558.8 "Sinbound Meltdown (x9)" duration 8.2
-563.6 "Dark Fire III" Ability { id: "9D54", source: "Oracle of Darkness" }
-563.6 "Unholy Darkness" Ability { id: "9D55", source: "Oracle of Darkness" }
-569.3 "Sinbound Meltdown" Ability { id: "9D2B", source: "Delight's Hourglass" }
-571.3 "Sinbound Meltdown (x9)" duration 8.2
-574.9 "Shadoweye" Ability { id: "9D56", source: "Oracle of Darkness" }
-574.9 "Dark Eruption" #Ability { id: "9D52", source: "Oracle of Darkness" }
-574.9 "Dark Water III" Ability { id: "9D4F", source: "Oracle of Darkness" }
-574.9 "Shadoweye" Ability { id: "9D56", source: "Oracle of Darkness" }
+544.2 "Dark Fire III/Unholy Darkness" Ability { id: "9D54", source: "Oracle of Darkness" }
+549.3 "Sinbound Meltdown 1 (x10)" Ability { id: "9D2B", source: "Delight's Hourglass" } duration 10.2
+554.0 "Dark Fire III/Dark Blizzard III/Unholy Darkness" Ability { id: "9D54", source: "Oracle of Darkness" }
+558.8 "Sinbound Meltdown 2 (x10)" Ability { id: "9D2B", source: "Delight's Hourglass" } duration 10.2
+563.6 "Dark Fire III/Unholy Darkness" Ability { id: "9D54", source: "Oracle of Darkness" }
+569.3 "Sinbound Meltdown 3 (x10)" Ability { id: "9D2B", source: "Delight's Hourglass" } duration 10.2
```

Comment:
`Sinbound Meltdown`'s source actor names can be stale, causing this sync to miss. It's not super important, I only noticed because I ran test_timeline on some pulls.

Running make_timeline on one of the pulls with stale names shows this:
```
549.2 "Sinbound Meltdown" Ability { id: "9D2B", source: "Oracle of Darkness" }
558.6 "Sinbound Meltdown" Ability { id: "9D2B", source: "Oracle of Darkness" }
569.1 "Sinbound Meltdown" Ability { id: "9D2B", source: "Delight's Hourglass" }
```